Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -885,6 +893,10 @@ bool AccessibilityNodeObject::computeIsIgnored() const
         // Only allow display:none / hidden-visibility node-only objects for canvas subtrees.
         return true;
     }
+
+    if (isTableCell())
+        return !isExposedTableCell();
```

Comment:
It's a bit unfortunate that we call `isTableCell()` twice here, but probably not a huge deal. One idea would be to have a method called `isIgnorableTableCell()` that returns false for `isTableCell()`, or true for no exposable ancestor table exists. What do you think?